test_duckdb_64bit.py

```
@@ -0,0 +1,269 @@
+    from src.database.sqlite_handler import SQLiteDatabase
+    print("  [OK] SQLiteDatabase imported")
+except ImportError as e:
+    print(f"  [NG] Cannot import SQLiteDatabase: {e}")
+    sys.exit(1)
+
+print("\n--- Database Operations Test ---")
+with tempfile.TemporaryDirectory() as tmpdir:
+    db_path = Path(tmpdir) / "test.duckdb"
+
+    config = {"path": str(db_path)}
+    db = DuckDBDatabase(config)
```

Ruff (0.14.14)

[error] Undefined name `DuckDBDatabase`

(F821)